lib/deploy.js

```diff
@@ -453,26 +456,23 @@ Deployer.prototype.deploy = function(options, svcname, callback) {
             function updateZKServers(nic, cb) {
                 var sapi = deployer.SAPI;
                 var log = deployer.log;
+                var metadata = {};
```

Comment:
Changes in this function are to only send `{metadata: {ZK_SERVERS: ...}}` to SAPI UpdateApplication, rather then the *entire* set of metadata for the "manta" app -- which is what it was doing before.